package.json

```diff
@@ -15,11 +15,14 @@
     "apm": "./bin/apm"
   },
   "scripts": {
-    "build": "node script/check-version.js && grunt",
-    "prepare": "grunt prepare",
+    "check-version": "node script/check-version.js",
+    "clean": "shx rm -rf lib bin/node_darwin_x64 && shx mkdir -p lib",
+    "lint": "coffeelint src spec",
+    "coffee": "coffee --compile --output lib src",
```

Comment:
This caused lines like this to be added at the top of every file in `lib/`:

`// Generated by CoffeeScript 2.5.1`

The old `coffee` task was equivalent to running `coffee --no-header --compile --output lib src `

I would recommend adding `--no-header` again.